react/Breadcrumb.tsx

```diff
@@ -53,46 +51,40 @@ class Breadcrumb extends Component<Props> {
         .replace(/[-\s]+/g, '-')
       categoryStripped += currentCategories.length === 1 ? '/d' : ''
       return {
-        name: categoryKey.toLowerCase(),
-        value: categoryStripped,
+        name: getCategoryName(categoryKey),
+        link: categoryStripped,
       }
     })
-  }
-
-  public render(): ReactNode {
-    const { term, categories } = this.props
+  }, [categories])
```

Comment:
`categories`  is an array, I think providing an array to `useMemo` makes it pointless because it will compare array pointers and always signal that the data changed, so `useMemo` will not have any effect.  Just like a PureComponent with an object on a prop.

But check for this on internet, estou tirando essa suposição da minha bunda :)